### ISSUE TITLE
Banner on membership sites

### ISSUE THREAD

**Issue by joyannnielsen** [Simplero/Roadmap#131]
To make membership sites and courses more personal/branded it would be GREAT to be able to add a banner image :-) Thanks.

**Comment by nickbruun** [comment]
Would this banner image be full width or centered? Where would you suggest it would be in the layout?

**Comment by joyannnielsen** [comment]
Hi Nick

 

Thanks for asking ☺

 

I would place the banner full width above the top nav, and set it up so that each course on the membership site has (the option of a) different banner image. See attachment. 

 

Another suggestion:

I am creating a course with 8 modules, and several lessons in each module. Would be great to be able to open/close the sidebar module menus, like this screenshot, having module 4 open/active: (doesn´t have to have graphics before the sub menu items). 

 

 

 

 

 

Bedste hilsner

Joyann Nielsen

 

Email: Joyann@fengshuiwebdesign.dk

Mobil: 4143 2001

Web: fengshuiwebdesign.dk

 

Facebook: fengshuiwebdesigndk

CVR: 27 35 51 88

Adresse: Nørregade 10, 8543 Hornslet

 

 

 

Fra: Nick Bruun <notifications@github.com>
Svar til: Simplero/Roadmap <reply@reply.github.com>
Dato: onsdag den 4. oktober 2017 kl. 21.22
Til: Simplero/Roadmap <Roadmap@noreply.github.com>
Cc: joyannnielsen <joyann@fengshuiwebdesign.dk>, Author <author@noreply.github.com>
Emne: Re: [Simplero/Roadmap] Banner on membership sites (#131)

 

Would this banner image be full width or centered? Where would you suggest it would be in the layout?

**Comment by joyannnielsen** [comment]
Hi Nick

Would you like all suggestions on the roadmap page? 
I have one more:

On the course page https://website-guides-for-lightworkers.simplerosites.com/courses/11-wordpress-startup-guide it would be great to have the module titles link to a module front page, showing the module description that is currently only showing op on the course front page. 

 

Maybe it´s just me, having this large course with traditional modules and lessons….? But intuitively I am looking for a link to each module when viewing the course front page.

 

I know that people use Simplero in many creative ways ;-) and I´ll do that too..

 

 

 

Happy thoughts,

Joyann Nielsen

 

Email: Joyann@lightworkerwebsite.academy

Mobil: 4143 2001

Web: lightworkerwebsite.academy

 

CVR: 27 35 51 88

Address:: Nørregade 10, 8543 Hornslet, Denmark

 

 

 

Fra: Nick Bruun <notifications@github.com>
Svar til: Simplero/Roadmap <reply@reply.github.com>
Dato: onsdag den 4. oktober 2017 kl. 21.22
Til: Simplero/Roadmap <Roadmap@noreply.github.com>
Cc: joyannnielsen <joyann@fengshuiwebdesign.dk>, Author <author@noreply.github.com>
Emne: Re: [Simplero/Roadmap] Banner on membership sites (#131)

 

Would this banner image be full width or centered? Where would you suggest it would be in the layout?

**Comment by nickbruun** [comment]
For now we won't be making modules separate pages for a number of reasons, but I'll definitely take your input on the banner image under consideration. I'm not sure exactly how we'd make it work with full width and then having the correct height across screen sizes. We'd have to squeeze the image on smaller screens – but what happens on larger screens?

**Comment by nickbruun** [comment]
It'd be great if you could submit requests as individually, as that makes it easier for us to have an overview (we'd like to be able to scan the list of requests on https://github.com/Simplero/Roadmap/issues to find individual ones.) Also note that screenshots don't work when replying by email.

**Comment by joyannnielsen** [comment]
Hi Nick

I work a lot in WordPress and to come across the banner image size challenge I add two different banner images, one 1920px wide for desktop and one 980 px wide for tablet/mobile, and I code the css as to show the mobile one on screens narrower than 980 px. 

 

But anything would work with banner I guess, if it´s easier to make it not-full-with, this could work too. 

 

 

Bedste hilsner

Joyann Nielsen

 

 

 

Fra: Nick Bruun <notifications@github.com>
Svar til: Simplero/Roadmap <reply@reply.github.com>
Dato: torsdag den 5. oktober 2017 kl. 19.15
Til: Simplero/Roadmap <Roadmap@noreply.github.com>
Cc: joyannnielsen <joyann@fengshuiwebdesign.dk>, Author <author@noreply.github.com>
Emne: Re: [Simplero/Roadmap] Banner on membership sites (#131)

 

For now we won't be making modules separate pages for a number of reasons, but I'll definitely take your input on the banner image under consideration. I'm not sure exactly how we'd make it work with full width and then having the correct height across screen sizes. We'd have to squeeze the image on smaller screens – but what happens on larger screens?

**Comment by JesperChristophersen** [comment]
I agree - a banner image above the navigation menu - like in spaces - would be brilliant.
